packages/jsapi-bootstrap/src/useObjectFetch.ts

```diff
@@ -31,7 +39,7 @@ export type ObjectFetchUpdateCallback<T = unknown> = (
 /** ObjectFetchManager for managing a subscription to an object using a VariableDescriptor */
 export type ObjectFetchManager = {
   /**
-   *
+   * Subscribe to an object using a variable descriptor.
```

Comment:
Just to be clear, this doesn't actually subscribe to the object, but subscribes to the function to fetch the object? It's still up to the consumer to actually fetch the object if they were using this hook.

Reply:
You might be thinking of charts built using `ChartBuilder`, I think that currently needs the fetch. Pretty sure I addressed that with my core plugins/xcomponent change if I recall correctly. 
My intent for this was useWidget is just for widgets. Could have another hook for `useTable`, `useFigure`, etc that is just built on top of `useObjectFetch`.
In any case, I wasn't going to replace the existing panels until Grizzly. This PR is intended for V+.